agent/php_nrini.c

```diff
@@ -255,6 +255,29 @@ static void check_ini_int_min_max(int* var, int minval, int maxval) {
   }
 }
 
+/*
+ * @brief error handling for strtoimax() functionality
+ *
+ * @param val_p   pointer to store the parsed value; remains undefined when
+ *                function returns NR_FAILURE
+ * @param str     string to parse
+ * @param base    base to be used for number conversion
+ * @return nr_status_t NR_SUCCESS || NR_FAILURE
+ */
+static nr_status_t nr_strtoi(int* val_p, const char* str, int base) {
+  int val;
+  char* endptr_p;
+  errno = 0;
+
+  val = (int)strtoimax(str, &endptr_p, base);
```

Comment:
I don't think the cast to `int` is needed here.
```suggestion
  val = strtoimax(str, &endptr_p, base);
```